includes/compat/trait-wc-stripe-subscriptions.php

```
@@ -373,6 +373,41 @@ public function scheduled_subscription_payment( $amount_to_charge, $renewal_orde
+		if ( empty( $charge_id ) ) {
+			return false;
+		}
+
+		$charge = WC_Stripe_API::retrieve( "charges/{$charge_id}" );
+
+		if ( empty( $charge ) || ! empty( $charge->error ) ) {
+			return false;
+		}
+
+		return isset( $charge->outcome->type ) && 'blocked' === $charge->outcome->type;
+	}
```

PHPMD (2.15.0)

[error] The method is_charge_blocked_by_radar is not named in camelCase. (undefined)

(CamelCaseMethodName)